docs/decex/domains/workflow.md

```diff
@@ -58,5 +57,13 @@ The domain block will then be carefully executed, one extrinsic at a time, allow
     
 ## Challenging Operators
     
-Any node who observes an Execution Receipt within any bundle for any consensus chain block that differs from what they produced locally has detected fraud. They will produce an extrinsic with a fraud proof to handle the fraud. If the fraud proof is valid, it will be included in the consensus chain, which will prune the Execution Receipt in question and all children from the block tree and slash all related operators. 
-    
\ No newline at end of file
+Any node who observes an Execution Receipt within any bundle for any consensus chain block that differs from what they produced locally has detected fraud. They will produce an extrinsic with a fraud proof to handle the fraud. If the fraud proof is valid, it will be included in the consensus chain, which will prune the Execution Receipt in question and all children from the block tree and slash all related operators. Currently, the challenge period is 256 domain blocks (~26 minutes).
+
+## Domain Block Rewards
+
+When a domain block is out of challenge period it is considered confirmed and can no longer be disputed. After a domain block is confirmed, the rewards for this block are applied as follows:
+
+- The total rewards of the confirmed block are added to the current epoch rewards for this domain. The total rewards of the block include all fees and tips of all of the transactions included in this block. The rewards are split equally among the operators pools who have previously submitted the Execution Receipt for this block and noted in the Operator Registry. The rewards do not affect the stake distribution yet. All the rewards will be auto-staked to the pools' stakes at the end of the current epoch. For more details on staking epochs, see the [Staking](/docs/decex/staking.md#staking-epochs) page.
```

Comment:
I think a graphic showing a timeline from transaction submission -> bundle -> execution -> confirmed would be helpful in making this very clear.

Reply:
Great idea. I will add it separately.